packages/fluentui/accessibility/src/behaviors/Breadcrumb/breadcrumbDividerBehavior.ts

```diff
@@ -0,0 +1,15 @@
+import { Accessibility } from '../../types';
+
+/**
+ * @specification
+ * Indicator is usually only visual representation and therefore is hidden from screen readers, unless 'aria-label' property is provided.
```

Comment:
This description does not match implementation, please check/update it.

Reply:
🙈my bad, copy-pasted and didn't check